### ISSUE TITLE
Testing different configuration for training the model.

### ISSUE THREAD

**Issue by khorrams** [khorrams/Carrene#1]
So far, we have come up with a model that assigns ICD-9 labels with f-score of around 50 percent. The baseline method (presented in the "Tagging patient notes with ICD-9 codes" paper) had reached the f-score of around 70 in its best configuration. The results that we are going to enhance are as follow (has run on Oct 2nd): 

+-------------+-------+
|    loss     | 0.495 |
+-------------+-------+
|  precision  | 0.684 |
+-------------+-------+
|   recall    | 0.39  |
+-------------+-------+
|   f_score   | 0.497 |
+-------------+-------+
|  accuracy   | 0.323 |
+-------------+-------+

The model has many hyperparameters that should be validated for different values. To name a few important parameters, number of LSTM layers, the maximum length of the words, number of the dense layers, model's optimizer algorithm, batch size, learning rate, etc. can be referred.

**Comment by khorrams** [comment]
This is a really time-consuming task which needs GPU computation power to boost the processing.
